docker-compose.yml

```diff
@@ -22,8 +22,8 @@ services:
   nginx-api:
     image: raster-foundry-nginx-api
     build:
-      context: ./nginx
-      dockerfile: Dockerfile.api
+      context: .
+      dockerfile: ./nginx/Dockerfile.api
     environment:
```

Comment:
I had to do more un-learning here. I thought that the context at the repo root would require docker to load the whole `app-backend/` (including `target/` dirs, which have a lot of stuff, some of it pretty large) tree into context. I've been burned by this before and thought it might be painful. But this time, docker doesn't seem to care about all that? Maybe docker got better 🤔